example2/index.html

```diff
@@ -0,0 +1,383 @@
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4"
+    crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.0/dist/bootstrap-toc.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" href="example.css">
+  <title>Skygear-SDK-JS examples</title>
+</head>
+
+<body>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+    crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm"
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.0/dist/bootstrap-toc.min.js"></script>
+  <script>
+    window.logP = function (el) {
+      return function (P) {
+        return (
+          P.then(function (res) {
+            el.value = JSON.stringify(res, null, 4)
+            el.style.color = '#222'
+          }).catch(function (err) {
+            el.value = JSON.stringify(err, null, 4)
+            el.style.color = 'red'
+          })
+        )
+      }
+    }
+  </script>
+  <script src="/bundle.js"></script>
+
+  <div class="container my-3" style="max-width: 850px">
+    <div class="row">
+      <div class="col-sm-9">
+        <!-- Current Info -->
+        <div>
+          <h2>Current Information</h2>
+          <div style="padding: 20px" class="border border-grey">
+            <div class="row info-row">
+              <div class="col">Endpoint</div>
+              <div id="info-endpoint" class="col-8"></div>
+            </div>
+            <div class="row info-row">
+              <div class="col">API Key</div>
+              <div id="info-api-key" class="col-8"></div>
+            </div>
+            <div class="row info-row">
+              <div class="col">Username</div>
+              <div id="info-username" class="col-8"></div>
+            </div>
+            <div class="row info-row">
+              <div class="col">Email</div>
+              <div id="info-email" class="col-8"></div>
+            </div>
+            <div class="row info-row">
+              <div class="col">Access token</div>
+              <div id="info-token" class="col-8" style="word-break: break-all;"></div>
+            </div>
+          </div>
+
+          <script>
+            function refreshUserInfo() {
+            $('#info-token')[0].innerText = skygear.auth.accessToken;
```

Comment:
Since we use the example as a quick start of using the JS SDK. In order to keep the example simple and easy to read, please avoid using 3rd party library in the SDK example functions. For example, if we only use jquery for finding the dom elements, maybe we just use document.getElementById and give element a clear and easy to understand id. Easy to read is important, as user will read the example to know how to use the api

I think it is ok to use library in the case that help make the ui better, for example `scrollspy`. If possible, we can group those layout setup functions in a js file. So we can reuse it in different pages and user doesn't need read it.